src/components/AzureMapFeature/AzureMapFeature.tsx

```diff
@@ -22,10 +25,35 @@ const AzureMapFeature = memo((props: IAzureMapFeature) => {
     }
   }, [])
 
+  useEffect(() => {
+    // Shape as Feature Wrapper for dynamically changes
+    if (createShape) {
+      createShape(featureRef)
+    }
+  }, [featureRef])
+
+  // Shape's methods
+  useEffect(() => {
+    if (shapeRef && setCoords) {
+      shapeRef.setCoordinates(setCoords)
+    }
+  }, [setCoords])
+
+  useEffect(() => {
+    if (shapeRef && setProperties) {
+      shapeRef.setProperties(setProperties)
+    }
+  }, [setProperties])
```

Comment:
We don't want to check this is somehow better ? like check if nested properties changed not only `ref`